module/VuFind/src/VuFind/Config/Upgrade.php

```diff
@@ -1470,16 +1470,16 @@ protected function extractComments($filename)
 
             // Is the current line a comment?  If so, add to the currentComments
             // string. Note that we treat blank lines as comments.
-            if (substr($trimmed, 0, 1) == ';' || empty($trimmed)) {
+            if (empty($trimmed) || str_starts_with($trimmed, ';')) {
```

Comment:
Any particular reason you left this as `empty` while changing other `empty` calls to `!== ''`?

Reply:
Just missed it. Fixed!